enatega-multivendor-web/.env.example

```
@@ -1,2 +1,3 @@
+## Local API in this repo runs on :8001
 NEXT_PUBLIC_SERVER_URL="http://localhost:8001/"
```

dotenv-linter (4.0.0)

[warning] [QuoteCharacter] The value has quote characters (', ")

(QuoteCharacter)